Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.yml

```diff
@@ -122,8 +122,8 @@ configuration:
   section: Connect
   advanced: true
 description: Exchange Web Services and Office 365 (mail)
-display: EWS v2
-name: EWS v2
+display: EWS_v2_2019_version
```

Comment:
we shouldnt change display name as the integration is compatible with 2013 and 2016 too

Reply:
oops just for testing, reverting